skype/skype-ps/skype/New-CsMobilityPolicy.md

```diff
@@ -572,24 +547,20 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
+
 This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).`
 
 ## INPUTS
 
-
-###  
 None.
 The New-CsMobilityPolicy cmdlet does not accept pipelined input.
 
 ## OUTPUTS
 
-###  
```

Comment:
Think the type `Microsoft.Rtc.Management.WriteableConfig.Policy.Mobility.Mobility` needs to be the header, but either way it can't be removed